MapboxNavigation/NavigationViewController.swift

```diff
@@ -758,7 +758,7 @@ extension NavigationViewController: StyleManagerDelegate {
 public extension NavigationViewController {
     @available(*, obsoleted: 0.1, renamed: "navigationService", message: "NavigationViewController no-longer directly manages a RouteController. See MapboxNavigationService, which contains a protocol-bound reference to the RouteController, for more information.")
     /// :nodoc: obsoleted
-    @objc public final var routeController: RouteController! {
+    @objc public final var routeController: LegacyRouteController! {
```

Comment:
This use of the now-deprecated LegacyRouteController class introduces build warnings, right? What’s our strategy for suppressing those warnings, which would show up in a CocoaPods-managed application project?

Reply:
These were deprecated 4 months ago. Can we remove them? They provide fix-its but apart from that, they just call `fatalError()`.